File: day 35 python/homework/hw3.py
# 4) ფუნქცია იღებს რიცხვს და თუ ის ლუწია, გაორმაგებს, წინააღმდეგ შემთხვევაში 3–ით გაიმრავლებს.



def if_Even(num):
    if num % 2 == 0:
        return num * 2
    else:
        return num * 3
print(if_Even(5))

File: day 58 Js/homework/hw3.js
// 3) დაწერეთ ფუნქცია სადაც კენტ რიცხვებს აიყვანთ კვადრატში და ლუწს კუბში ხოლო თუ რიცხვი ნულის ტოლია გამოიტანეთ "Zero".


function pow1(num){
    if ( num > 0 &&  num % 2 ==0 ){
        return num ** 2
    } else if ( num > 0 && num % 2 != 0){
        return num ** 3
    } else{
        return "Zero"
    }

}

console.log(pow1(10))